ui.tests/test-module/specs/formcontainer.spec.js

```diff
@@ -255,7 +255,7 @@ describe('Page/Form Authoring', function () {
             const pagePath = "/content/forms/af/core-components-it/blank.html";
 
             beforeEach(function () {
-                cy.previewForm(pagePath);
+                cy.previewForm(pagePath, {"params" : ["formRenderingInsideEmbedContainer=''"]});
```

Comment:
Why do you need this params here ? Can you remove this. This param is added OOTB inside embed

Reply:
Removed.